resources/newssite/news-next/src/workload-test.mjs

```diff
@@ -0,0 +1,41 @@
+import { BenchmarkTestStep, BenchmarkTestSuite, BenchmarkSuitesManager, forceLayout, getElement } from "speedometer-utils/workload-testing-utils.mjs";
+
+export function initWorkload() {
+    window.benchmarkSuitesManager = new BenchmarkSuitesManager(window.name, [
```

Comment:
I guess we could actually just pass this object as a parameter into the call to `connectToBenchmark` now that we're just calling the function (instead of sticking it on the window object). So just have this module call connectToBenchmark directly instead of sticking it on benchmarkSuitesManager and then having the App object call both. WDTY @flashdesignory ?

Reply:
done!